task/deprecated-image-check/0.1/README.md

```diff
@@ -0,0 +1,18 @@
+# deprecated-image-check task
+
+## Description:
+Various (deprecated) label checks
```

Comment:
The deprecated image check is different from the deprecated label checks.
[This check](https://redhat-appstudio.github.io/docs.stonesoup.io/Documentation/main/concepts/testing_applications/sanity_tests.html#_deprecated_image_checks) verifies that the container image isn't built from a repository which is marked as 'Deprecated' in RedHat COMET database (Pyxis). Deprecated images are no longer maintained and will accumulate security issues without releasing a fixed version.

Reply:
The new description is fine, but I think it would be good to also a blurb like this to help with understanding what the task does behind the scenes:
```
The the check verifies that the container image isn't built from a repository which is marked as 'Deprecated' in RedHat COMET database (Pyxis). If the image isn't found in the database, the check is skipped.
```